features/step_definitions/buyers_management/user_steps.rb

```diff
@@ -19,13 +19,13 @@
 end
 
 Then /^I should not see button to delete buyer (user "[^"]*")$/ do |user|
```

Comment:
I ever wonder the use of those I should NOT see steps

They are old but we should probably strie for getting rid of them (later)